mbed/mbed.py

```diff
@@ -328,7 +325,7 @@ def cleanup():
                     shutil.rmtree(fl)
 
     def clone(url, path=None, depth=None, protocol=None):
-        m = Bld.isurl(url)
+        m = Bld.isvalidurl(url)
```

Comment:
Does this mean that cloning a repo from the filesystem is not supported?

Reply:
Bld is emulation of SCM. It only exists as a function of the mbed Website.

Reply:
Personally, I would say that Bld is not SCM, but is supported by Mbed CLI. just my 2c.